stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -488,14 +494,14 @@ public struct ${Self}<Pointee>
   ///   - count: The number of consecutive copies of `newValue` to initialize.
   ///     `count` must not be negative. The default is `1`.
   @_inlineable
-  public func initialize(to newValue: Pointee, count: Int = 1) {
+  public func initialize(repeating repeatedValue: Pointee, count: Int = 1) {
```

Comment:
Please update docs when the API changes—in this case, the name of the method is now `intialize(repeating:count)` and the first parameter name has changed from `newValue` to `repeatedValue`.

Reply:
Thanks for taking care of these!